content/about/pricing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Pricing
+weight: 4
+---
+
+Workers is now free to enable for all and are subject to some limits. 
+
+## Unlimited 
+To avoid the limits and enable KV, you can subscribe to the Workers Unlimited Plan for **$5 USD per month for a account**. Workers Unlimited plan is separate from any Cloudflare plan (Free, Professional, Business) you may have.
+
+If you are an Enterprise customer, reach out to your account team to confirm pricing details.
+
+## Limits 
+
+To protect our systems against abuse there are surge limits in place for domains without subscriptions. 
```

Comment:
s/domains/accounts/ ?

Also, rather than "surge limits", consider using language that maps directly to the language in the tables below, e.g. "burst limits".